### PR TITLE
Deck builder refactor part4

### PR DESCRIPTION
**Pull request by lily-pytel**
Last PR for the night. That slider is copied clean off the internet - don't judge.

Deck builder page with "view cards" selected:

image

Deck builder page with "view filters" selected:

image

Slides used in the decks page too:

image

I tell you, this really messes with Cypress' head. You have to tell it to go back and forth constantly.

### CODE REVIEW

File: e2e/cypress/page-objects/all.js [hunk 1]
```diff
@@ -32,5 +32,9 @@ export default {
 
   // decks page
   deckFactionsPicker: '[data-cy="deckFactionsCell"] img',
-  deckEssencePicker: '[data-cy="deckEssenceCell"]'
+  deckEssencePicker: '[data-cy="deckEssenceCell"]',
+
+  // slider
+  leftLabel: '[data-cy="sliderLeftLabel"]',
+  rightLabel: '[data-cy="sliderRightLabel"]'
```
Comment (alexmiddeleer) [inline e2e/cypress/page-objects/all.js]: I suggest `leftLabel` => `leftSlider` or make a `moveSliderLeft` fn. Could nix the comment as well.

File: next/pages/deck-builder.js [hunk 2]
```diff
@@ -97,38 +44,38 @@ function DeckBuilderPage() {
...
+              setCardManaCosts={setCardManaCosts}
+              setSupertypes={setSupertypes}
+              setCardRarities={setCardRarities}
+            />
           )}
-          {currentTab === 'Powers' && (
-            <div className="collection" data-cy="deckBuilderPaths">
-              <AllPowers onPowerClick={onPowerClick}></AllPowers>
-            </div>
+          {!viewFilters && (
```
Comment (alexmiddeleer) [inline next/pages/deck-builder.js]: I think this way if doing if-else is slightly more readable but /shrug (from react docs)
image

File: e2e/cypress/page-objects/all.js (hunk 1 above)
Comment (lily-pytel) [reply e2e/cypress/page-objects/all.js]: Changed

File: next/pages/deck-builder.js (hunk 2 above)
Comment (lily-pytel) [reply next/pages/deck-builder.js]: You're totally right. It was late.

File: e2e/cypress/page-objects/all.js (hunk 1 above)
Comment (lily-pytel) [reply e2e/cypress/page-objects/all.js]: Although I like the comments on this page

**Pull request merged**